packages/flutter/lib/src/material/adaptive_text_selection_toolbar.dart

```diff
@@ -211,6 +211,7 @@ class AdaptiveTextSelectionToolbar extends StatelessWidget {
             return localizations.pasteButtonLabel;
           case ContextMenuButtonType.selectAll:
             return localizations.selectAllButtonLabel;
+          case ContextMenuButtonType.delete:
```

Comment:
So here I think this should return `localizations.deleteButtonTooltip`.  It looks like you're looking up the localization inline [in spell_check_suggestions_toolbar.dart](https://github.com/flutter/flutter/pull/114460/files#diff-1dd45316aef4f75e739df65a95c6c1877407fdf881ed8d4383a12aaedae7b2f7R87).  The benefit of defining it here is that users can build their own delete button without worrying about the label.